watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -1110,15 +1110,6 @@ def build(self):
             ("R19", "Liq", "X_I"): 0,
         }
 
-        s_ic_rxns = ["R5", "R6", "R10", "R11", "R12"]
-
-        for R in s_ic_rxns:
-            self.rate_reaction_stoichiometry[R, "Liq", "S_IC"] = -sum(
-                self.Ci[S] * self.rate_reaction_stoichiometry[R, "Liq", S] * mw_c
-                for S in Ci_dict.keys()
```

Comment:
So what I'm working on now (in a separate PR) is removing this for loop (this is something Alejandro, Chenyu, and I discussed earlier) and making sure the Petersen matrix entries for S_IC are being computed correctly. For example, there are some places where the value for `Ci["X_su"]` is being hard-coded rather than called from the dictionary

Reply:
This shouldn't be a functional change in any way. It's essentially just incorporating this for loop into the matrix directly - which is consistent with what Chenyu and I did for modified ADM1

Reply:
I see. I put this for loop in in the initial formulation of ADM1 (https://github.com/watertap-org/watertap/pull/627/commits/bf942b39808f267e44130afc58f39fa40b1e82c7) thinking it would be more efficient and cleaner. Can you remind me why this was even consider an issue? Is it more an issue of consistency?

Reply:
After working on the new PR (and discussing with Alejandro behind-the-scenes), I see why you put in the for loop and it might make more sense to just keep it in. But yes, the original issue Chenyu and I saw with it was just consistency. So I think the solution Alejandro and I reached is that we should just delete the 4 keys in Ci_dict that are unused.

Reply:
Wait- but are they actually unused, or are they just being used under a different parameter/variable name?

Reply:
The latter, I believe

Reply:
The change affected a tested value of the matrix (which shouldn't be affected by pressure in any way)